### PR TITLE
Add installation procedure in README.md

### PR DESCRIPTION
**Pull request by tanishi**
Add installation procedure in README.md for user

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,6 +37,10 @@ To build locally with Drone:
 - Install the [Drone command line tools][dronecli].
 - Run `drone exec` from within the root directory of the govmomi repository.
 
+## Installation
+
+`go get github.com/vmware/govmomi`
+
```
Comment (dougm) [inline README.md]: Thanks, can you add the `-u` flag and use the same markup as: https://github.com/vmware/govmomi/tree/master/govc#source

Comment (tanishi) [reply README.md]: Thanks review!

**Comment by tanishi** [comment]
I modified. Do I understand it properly?
Please review:bow:

**Comment by dougm** [comment]
:+1:

**Pull request merged**